packages/compiler/lib/std/decorators.tsp

```diff
@@ -277,13 +277,6 @@ extern dec maxValueExclusive(target: numeric | ModelProperty, value: valueof num
  */
 extern dec secret(target: string | ModelProperty);
 
-/**
- * Mark this operation as a `list` operation for resource types.
```

Comment:
Did we get this removal approved?  Since it's deprecated, I doubt this is controversial